src/main/java/com/adq/jenkins/xmljobtodsl/InitialArgumentsHandler.java

```diff
@@ -10,10 +10,9 @@
 import javax.xml.parsers.ParserConfigurationException;
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.security.Key;
+import java.util.*;
```

Comment:
If I see correctly, no more imports were added in this file. Why is the wildcard import a better solution here?

Reply:
Not sure, a teammate added this. I can try removing it and make sure things still look good.

Reply:
Ok so according to my IDE this line supports Map, HashMap, containsKey etc so that's probably why the wildcard solution is used. How it was working before I'm not totally sure